src/js/component.js

```diff
@@ -220,28 +220,35 @@ class Component {
    * @return {string}
    *         The localized string or if no localization exists the english string.
    */
-  localize(string) {
+  localize(string, tokens, defaultValue = string) {
```

Comment:
Does it ever make sense to make the default value anything other than the string that was passed in for localization though?